src/promptflow/promptflow/_sdk/_start_trace.py

```diff
@@ -26,8 +31,10 @@ def start_trace():
 
     Note that this function is still under preview, and may change at any time.
     """
-    # TODO: PFS liveness probe
-    pfs_port = get_port_from_config()
+    pfs_port = get_port_from_config(create_if_not_exists=True)
+    _start_pfs_in_background(pfs_port)
+    # Start a liveness probe in a new thread
+    threading.Thread(target=_liveness_probe, args=(pfs_port,), daemon=True).start()
```

Comment:
timeout for this action?

Reply:
interval=60

Reply:
I think you mean the retry interval here. 
What I feels strange is why here is a while true probe here?

Reply:
removed while true